scss/login.scss

```diff
@@ -0,0 +1,225 @@
+body.modal-open {
+  #main-outlet {
+    min-height: 100vh;
+  }
+}
+
+#discourse-modal .create-account .modal-body {
+  max-height: none !important;
+}
+
+.d-modal.forgot-password-modal,
+.d-modal.create-account,
+.d-modal.create-account-modal,
+.d-modal.login-modal {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 1300;
+  overflow: auto;
+
+  // Background
+  &:before {
+    content: "";
+    height: 100vh;
+    width: 100vw;
+    background: $background-gradient;
+    position: fixed;
+    z-index: -1;
+    animation: none;
+  }
+
+  // Signup form
+  .login-form {
+    position: relative;
+    display: flex;
+    overflow: visible;
+    flex: 1;
+
+    &:after,
+    &:before {
+      display: none;
+    }
+
+    form {
+      width: 100%;
+      padding: 0 24px;
+      max-height: none;
+
+      table {
+        width: 100%;
+
+        td {
+          padding: 0;
+          width: 100%;
+          margin: 0 auto;
+        }
+      }
+    }
+  }
+
+  // Login and reset password forms
+  .forgot-password-modal-form,
+  #login-form {
+    margin: 0 auto;
+    padding: 0 24px;
```

Comment:
Try always to favor the usage of `rem`. In this case `1.5rem`.
Why? Because in that way the user preferences are respected.